recipes/freebayes/meta.yaml

```diff
@@ -24,16 +25,16 @@ requirements:
     - htslib
     - parallel
     - meson
+    - python
     - ninja
     - bc
     - samtools
     - vcflib
     - pkg-config #needed for MacOS build
     
   run:
-# Freebayes does not require vcfpy, but it does require python 3.  I don't know how to require python 3 but not force python 3.9  
-    - vcfpy
     - htslib
+    - python
```

Comment:
Would a `- python >=3` also work?

Reply:
So, I had this previously, but it forces a pinning to python 3.9 (https://github.com/bioconda/bioconda-recipes/pull/27011#issuecomment-789970608).  

I tried using different iterations of `python >=3` in the run section, and it would always specifically build for python 3.9.  I originally solved this by requiring a small python only library (`vcfpy`) that could properly limit `python >=3`.  This seemed functional but not optimal as it was not a requirement of freebayes.

Looking through comments on Gitter and asking @dpryan79, `skip: True  # [py27]` seemed like the best solution.